pyproject.toml

```diff
@@ -0,0 +1,70 @@
+[tool.mypy]
```

Comment:
Can you just add a directory instead of each file? something like seems to work according to https://stackoverflow.com/a/78480102/6508131:

```python
exclude = [
"examples/",
"doc",
"pulp/apis",
"pulp/solverdir",
]
```